### DISCUSSION TITLE
Can we preload canvas with an existing svg file?

### DISCUSSION THREAD

**Discussion by aftabaig** [vinothpandian/react-sketch-canvas#57] [Q&A] [answered]
I have a requirement where I have a town plot map and I want to draw a polygon against each plot to show the border of each plot

**Comment by vinothpandian** [comment, marked as answer]
You can add the image to `backgroundImage` property as a background then allow users to draw on top of it. But this library doesn't support polygons. I would suggest you use another feature-rich library like [tldraw](https://www.npmjs.com/package/@tldraw/tldraw)
